plugins/parser/multiline.md

```diff
@@ -23,20 +23,21 @@ See [Parse Section Configurations](/configuration/parse-section.md).
 Specifies the regexp pattern for the start line of multiple lines. Input plugin
 can skip the logs until `format_firstline` is matched. Default is `nil`.
 
-If `format_firstline` is not specified, input plugin should store
-unmatched new lines in temporary buffer and try to match buffered logs
+If `format_firstline` is not specified, the input plugin should store the
+unmatched new lines in the temporary buffer and try to match the buffered logs
 with each new line.
 
 
-### `formatN`
+### `formatN` (required)
 
-| type   | default            | version |
-|:-------|:-------------------|:--------|
-| string | required parameter | 0.14.0  |
+| type   | default | version |
+|:-------|:--------|:--------|
+| string | `nil`   | 0.14.0  |
 
-Specifies the regexp patterns. For readability, you can separate regexp patterns
-into multiple `regexpN` parameters, See Rails Log example below. These patterns
-are joined and construct a regexp pattern with multiline mode.
+Specifies the regexp patterns. For readability, you can separate the regexp
+patterns into multiple `regexpN` parameters. See the Rails Log's example below.
```

Comment:
This is original typo but `formatN` is correct, not `regexpN`

Reply:
So, `regexpN` should be **patterns into multiple `formatN` parameters**, right?

Reply:
Yes